tests/Internal/Codebase/InternalCallMapHandlerTest.php

```diff
@@ -767,12 +719,12 @@ public function assertEntryReturnType(ReflectionFunction $function, string $entr
         } else {
             $expectedType = $function->getReturnType();
         }
-        if ($expectedType === null) {
-            $this->assertSame('', $entryReturnType, 'CallMap entry has incorrect return type');
-            return;
-        }
 
-        $this->assertTypeValidity($expectedType, $entryReturnType, true, 'CallMap entry has incorrect return type');
+        if ($expectedType !== null) {
+            $this->assertTypeValidity($expectedType, $entryReturnType, true, 'CallMap entry has incorrect return type');
+        } else {
+            $this->assertNotEmpty($entryReturnType);
```

Comment:
I'm not sure if this is ever true or if the parse checks above catch this, but we should define a return type at least.